### DISCUSSION TITLE
SVG vs. PNG?

### DISCUSSION THREAD

**Discussion by SamuraiCrow** [GCC-Tiny/.github#2] [General]
@HSorensen I see you made a PNG diagram for part3.  I took the initiative to hand-convert it to SVG for resolution independence and to make it more future-proof.  It's 15k vs. the 10k of the original PNG.  Which should we use?  The SVG format should be supported by most browsers nowadays and I didn't put any interactivity in it so it should be just as secure as a PNG.  

## Original:
![original](https://github.com/SamuraiCrow/SVG-icons/blob/master/part03_lexer-input.png?raw=true)

## Vector:
![vector](https://raw.githubusercontent.com/SamuraiCrow/SVG-icons/master/part03_lexer-input.svg)

**Comment by HSorensen** [comment]
That's great. I just downloaded the image that was provided. This is obviously better

**Comment by HSorensen** [comment]
Also just change the reference and merge to Main branch.

**Reply by HSorensen** [reply]
@SamuraiCrow will you replace the png file with this svg file, and adjust the image reference?

**Reply by SamuraiCrow** [reply]
Ok.  I can do that easily enough.  Thanks for the reminder.

**Reply by SamuraiCrow** [reply]
OK, it's done.  There appears to be some space embedded in the SVG but at least it's committed to the repo.
